tsconfig.json

```diff
@@ -22,7 +21,6 @@
     "useUnknownInCatchVariables": false,
     "noUnusedParameters": true,
     "forceConsistentCasingInFileNames": true,
-    "suppressImplicitAnyIndexErrors": true,
```

Comment:
[deprecated](https://devblogs.microsoft.com/typescript/announcing-typescript-5-5-rc/#disabling-features-deprecated-in-typescript-50) since TypeScript 5.0 and has no any effect since 5.5